packages/fxa-settings/src/pages/Signin/index.tsx

```diff
@@ -373,7 +373,7 @@ const Signin = ({
   const cmsInfo = integration.getCmsInfo();
 
   return (
-    <AppLayout integration={integration}>
+    <AppLayout cmsInfo={cmsInfo}>
```

Comment:
Just a tiny syntactic sugar nit, `cmsInfo={cmsInfo}` can be `{...{cmsInfo}}` in several places. 🤷‍♀️